### PR TITLE
Documentation maintenance. 

### PR DESCRIPTION
**Pull request by mjhenkes**
### Summary

Clean up a bunch of small doc problems. Fix links, update content, re-organize some things. I'm sure more content could be added but this just maintains the doc we have.

### Additional Details

Thank you for contributing to Terra.
@cerner/terra

### CODE REVIEW

File: src/terra-dev-site/about/contributing.d/DeveloperWorkflow.e.about.mdx [hunk 1]
```diff
@@ -53,7 +53,7 @@ The following guide covers the development workflow for the terra monorepos that
 * [terra-framework](https://github.com/cerner/terra-framework)
 * [terra-clinical](https://github.com/cerner/terra-clinical)
 
-Components, along with their docs and tests are organized as individual packages that share similar tooling. You can read more about the [directory structure of these components here](https://github.com/cerner/generator-terra-module#generated-package).
+Components, along with their docs and tests are organized as individual packages that share similar tooling.
```
Comment (tbiethman) [inline src/terra-dev-site/about/contributing.d/DeveloperWorkflow.e.about.mdx:56]:
```suggestion
Components, along with their docs and tests, are organized as individual packages that share similar tooling.
```

**Review by tbiethman — APPROVED** [review]

**Review by StephenEsser — APPROVED** [review]

**Review by benbcai — APPROVED** [review]

**Review by ryanthemanuel — APPROVED** [review]

File: GOVERNANCE.md [hunk 2]
```diff
@@ -40,7 +40,6 @@ The maintainers are individuals who are recognized by merit and have a demonstra
 The current Maintainers on the project are:
 
 - [Dave Kasper](https://github.com/dkasper-was-taken)
-- [Emily Rohrbough](https://github.com/emilyrohrbough)
```
Comment (emilyrohrbough) [inline GOVERNANCE.md:43]: 😢

**Review by neilpfeiffer — APPROVED** [review]
UX +1

File: GOVERNANCE.md (hunk 2 above)
Comment (neilpfeiffer) [reply GOVERNANCE.md:43]: 😢

File: src/terra-dev-site/about/contributing.d/ContributionGuidelines.a.about.mdx [hunk 3]
```diff
@@ -30,14 +30,14 @@ For larger contributions the Terra team needs to be consulted and included in th
...
-
-To ensure there are no surprises on PRs and the resulting component, contributors are expected to have regular syncs with the allocated Terra UX and Engineers. This can include periodic meetings and/or attending scrum.
-
 ### Tech design
 
 Before any other steps a contributor must provide a brief tech design outlining proposed prop APIs, responsive and accessibly considerations. This must be attached to the github issue.
 
+### Contributor Expectations
+
+To ensure there are no surprises on PRs and the resulting component, contributors are expected to have regular syncs with the allocated Terra UX and Engineers. This can include periodic meetings and/or attending scrum.
```
Comment (neilpfeiffer) [inline src/terra-dev-site/about/contributing.d/ContributionGuidelines.a.about.mdx:39]: 👍

File: src/terra-dev-site/about/contributing.d/ContributionGuidelines.a.about.mdx [hunk 4]
```diff
@@ -32,7 +32,7 @@ For larger contributions the Terra team needs to be consulted and included in th
 
 ### Tech design
 
-Before any other steps a contributor must provide a brief tech design outlining proposed prop APIs, responsive and accessibly considerations. This must be attached to the github issue.
+The first step is to log a github issue that includes a brief tech design outlining proposed prop APIs, responsive and accessibly considerations. It is crucial that contributors remain in sync with terra team standards and expectations to avoid expensive rework late in the contribution.
```
Comment (neilpfeiffer) [inline src/terra-dev-site/about/contributing.d/ContributionGuidelines.a.about.mdx:35]: 👍

**Pull request merged**